src/components/Alert.jsx

```diff
@@ -1,6 +1,44 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
+import Note from './Note';
+import { SetFieldValues } from "../utilities/FormHelpers";
+
+
+
+export const getAlertMessage = (props) => {
```

Comment:
any of the functions in this component that are not the actual component but are used by other components shoudl be moved to a utility probably called `AlertHelpers`.